.funcignore

```diff
@@ -0,0 +1,5 @@
+.git*
+.vscode
+local.settings.json
+test
+.venv
```

Comment:
nitpick: newline

Reply:
Ah right - these files were auto-generated by VSCode but agreed we should enforce EOF new lines in general (VSCode also creates `^M`s at the end of each line that I would rather do without...). I can add an explicit `Black` config here which would ensure these are present